fortio_main.go

```diff
@@ -145,6 +145,10 @@ func main() {
 	if err != nil {
 		usage("Unable to extract percentiles from -p: ", err)
 	}
+	filesList, err := results.ParseArgsForResultFiles(os.Args)
```

Comment:
this will assume usage to be:

```
fortio merge file1.json file2.json
```

caveat, it won't honor the `-json` flag currently without some special casing.
i will add support for `-` and `-a` before marking PR ready